packages/@okta/vuepress-site/docs/api/resources/system_log/index.md

```diff
@@ -557,7 +557,7 @@ The table below shows 18 events produced from 13 transactions over 6 different s
 |                             | Wij-564YuniRd9yTmWHoaQAAAII   | 0cc6f4c8-9b91-4a70-b5c4-09d6ad159d32   | mim.checkOSXAccessEligibility.true            | *blank*                               |
 |                             | Wij-2q4YuniRd9yTmWHjRAAAADA   | 92606da8-7eeb-4ad7-8ffb-502dd0ec64cc   | user.authentication.sso                       | User single sign on to app            |
 | *null*                      | Wm@-R2s5lEMbNIB03krtvAAACyo   | 566671be-ec0b-400d-ad2e-6fc73ed12fb1   | user.session.start                            | User login to Okta                    |
-
+{.table-word-break}
```

Comment:
?

Reply:
This is part of the markdown-it-attrs (https://github.com/arve0/markdown-it-attrs)  It applies a class to the table